migration/src/test/java/com/github/gfx/android/orma/migration/test/ManualStepMigrationTest.java

```diff
@@ -151,7 +151,7 @@ public void testIdempotenceWithNop() throws Exception {
     public void upgradeFull() throws Exception {
         migration.upgrade(db, 1, 100);
 
-        assertThat(migration.fetchDbVersion(db), is(16));
+        assertThat(migration.fetchDbVersion(db), is(100));
```

Comment:
Because calling ` migration.upgrade(db, 1, 100);`, the db version must be `100`. The line was my mistake!